modules/bibauthorid/lib/bibauthorid_dbinterface.py

```diff
@@ -790,39 +790,43 @@ def get_all_paper_data_of_author(pid):
     return _select_from_aidpersonidpapers_where(select=['personid', 'bibref_table', 'bibref_value', 'bibrec', 'flag'], pid=pid)
 
 
-def get_papers_of_author(pid, claimed_only=False, include_rejected=False):  # get_all_paper_records
+def get_papers_of_author(pid, include_claimed=True, include_unclaimed=True, include_rejected=False):   ### get_all_paper_records
```

Comment:
@petros-ioannidis  You completely changed the behaviour of the function -> previously it was only returning bibrecs, now it is returning everything.

Behaviour of `ignore_unclaimed` is wrong. `flag>-2 and flag<2` is never met.

You removed comments, and decided that you won't put any explanation. It seems that a person who developed this function before (maybe you) thought that it is not self-explanatory enough. Your code is not more readable than the previous one, so why didn't you write new comments?

There was a mode of this function. where you could ask for everything which was not rejected by authors. You removed it, I don't see a reason to do that.